Setting/Factions/CADMUS (Collective for the Advancement of Directed Morphogenetic Synthesis).md

```diff
@@ -0,0 +1,38 @@
+---
+tier: "5"
+allies:
+  - "[[Intership Council]]"
+  - "[[Navigation]]"
+  - Ship Defense Corps
+  - Scrappers
+enemies:
+  - "[[The Coven]]"
+demonym: cadmus
+aliases:
+  - CADMUS
+---
+#faction
+
+## Background
+Once known as the [[Foundation For Municipal Health]], this organization was borne from the need to supply medical care to generations of people. Over population becoming a concern the organization shifted focus from conventional medicine and extending human life, and focused on more eldritch research. There are extra test subjects afterall.. On the discovery of a synthetic mutagenic fungus, there was extensive research in its use in augmenting human capabilities and its side effects. Some seek immortality so they may experience what awaits in orbit of the [[Final Star]], while others examine unlocking the full potential of the human brain and cybernetics
+
+[[The Coven]] opposes their silo. Their knowledge belongs to the people, and bears wider use and experimentation. They extensively use what scraps of secrets they have uncovered, primarily from the sole few who have 'defected' to their cause.
+
+## Turf
+The location of all parts of [[The Academy]] are not common knowledge, hidden from the public to avoid fear of what their work comprises and to reduce reprisal. These include residences, although some choose to reside in the [[Garden District]]
+
+## NPCs
+[[Dr. Xavier Baleford]] - Working on the secret for immortality, a cessation of aging entirely. Initial trials have been deemed safe enough that he has seen a small cohort survive abnormally long. His research is nearly at a breakthrough.
+
+[[Dr. Angeline Angleheart]] - Working on a cure for [[Psychogenic Encephalopathy]], however the lab is about to leak, what escapes into the population?
+
+[[Dr. Aaron Mudd]] Someone who is working on naturally grown manipulators and augmentations, who needs an extra hand when you have three!
+## Notable Assets
+High security laboratory for studying hazardous biologicals, and an industrial euthanasia & reclamation system for test subjects..
+## Situation
+
+## Clocks
+
+`clock 3 / 10` Lab leak
+
+`clock 5/9` Immortality discovered
```

Comment:
Immortality or sufficiently extended lifetime and overpopulation as a threat to the stability of the ship seemed like a potentially interesting challenge. Here I am thinking that once it is 'discovered' there is risk of people finding out about it and/or risking groups trying to use it. Using it could even come at a great cost..

But also I could see this being a total mess to run in game haha

Reply:
It does feel like an odd thing to focus research on given the problems they have. Maybe it could be our vampirism?

Also curious if there might be other clocks here (people need to eat less, breathe less, etc)

Reply:
Yeahh.. I think there is an element of self serving vs community serving here I was excited about, of immortality for the very few, vs strength of the broader community and letting the generations play out naturally. But not extremely attached either. I think research on metabolic efficiency would be a more realistic area for research.

Reply:
I do like the the vampire analog, making the maintenance of immortality abhorrent and impractical for more than a few individuals would potentially help it sit in the setting better. 

Also great call on clocks that spell doom if research isn't complete, I'll add one or two there.